packs/data/pathfinder-bestiary-3.db/draconal-yellow.json

```
@@ -0,0 +1,2691 @@
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [],
+                "slug": null,
+                "source": {
+                    "value": ""
+                },
+                "traits": {
+                    "custom": "",
+                    "rarity": "common",
+                    "value": [
+                        "divine",
+                        "evocation"
+                    ]
+                },
+                "trigger": {
+                    "value": ""
+                },
+                "weapon": {
+                    "value": ""
+                }
+            },
+            "img": "systems/pf2e/icons/actions/TwoActions.webp",
+            "name": "Breath Weapon",
+            "sort": 2300000,
+            "type": "action"
+        },
+        {
+            "_id": "br8FCwfVCLAXisnU",
+            "data": {
+                "actionCategory": {
+                    "value": "offensive"
+                },
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p>Draconals embody the core value of wisdom, but every path to wisdom and enlightenment differs, so draconals' powers vary by temperament and coloration. A draconal's scale color determines their associated energy damage type.</p>\n<p>They gain resistance 15 to that damage type, deal 4d6 extra damage of that type with their Strikes, and deal 21d6 damage of that type with their breath weapon. They gain additional wisdom in the form of the basic and advanced domain spells from one domain, chosen from four domains available to draconals with their scale color.</p>\n<hr />\n<p><em>Note: Choose the domain powers from the available domains according to the Draconal's scale color and add them to the \"Champion Focus Spell\" spellcasting entry.</em></p>\n<ul>\n<li><em><strong>All colors</strong> @Compendium[pf2e.domains.Air Domain]{Air Domain} @Compendium[pf2e.domains.Healing Domain]{Healing Domain}</em></li>\n<li><em><strong>Black</strong> @Compendium[pf2e.domains.Fire Domain]{Fire Domain} @Compendium[pf2e.domains.Luck Domain]{Luck Domain}</em></li>\n<li><em><strong>Green</strong> @Compendium[pf2e.domains.Nature Domain]{Nature Domain} @Compendium[pf2e.domains.Water Domain]{Water Domain}</em></li>\n<li><em><strong>Red</strong> @Compendium[pf2e.domains.Protection Domain]{Protection Domain} @Compendium[pf2e.domains.Sun Domain]{Sun Domain}</em></li>\n<li><em><strong>White</strong> @Compendium[pf2e.domains.Freedom Domain]{Freedom Domain} @Compendium[pf2e.domains.Repose Domain]{Repose Domain}</em></li>\n<li><em><strong>Yellow</strong> @Compendium[pf2e.domains.Earth Domain]{Earth Domain} @Compendium[pf2e.domains.Luck Domain]{Luck Domain}</em></li>\n</ul>"
```

```suggestion
                    "value": "<p>Draconals embody the core value of wisdom, but every path to wisdom and enlightenment differs, so draconals' powers vary by temperament and coloration. A draconal's scale color determines their associated energy damage type.</p>\n<p>They gain resistance 15 to that damage type, deal 4d6 extra damage of that type with their Strikes, and deal 21d6 damage of that type with their breath weapon. They gain additional wisdom in the form of the basic and advanced domain spells from one domain, chosen from four domains available to draconals with their scale color.</p>\n<hr />\n<p><em>Note: Choose the domain powers from the available domains according to the Draconal's scale color and add them to the \"Champion Focus Spell\" spellcasting entry.</em></p>\n<ul>\n<li><em><strong>All colors</strong> @Compendium[pf2e.domains.Air Domain]{Air Domain} @Compendium[pf2e.domains.Healing Domain]{Healing Domain}</em></li>\n<li><em><strong>Yellow</strong> @Compendium[pf2e.domains.Earth Domain]{Earth Domain} @Compendium[pf2e.domains.Luck Domain]{Luck Domain}</em></li>\n</ul>"
```
Remove the colors that aren't All or Yellow

---

Done - for the yellow draconal and also for all other variants within their pull requests.